android/app/src/main/java/app/organicmaps/Framework.java

```diff
@@ -181,27 +190,10 @@ public static void setSpeedCamerasMode(@NonNull SettingsPrefsFragment.SpeedCamer
 
   public static native String nativeGetGe0Url(double lat, double lon, double zoomLevel, String name);
 
-  public static native String nativeGetAddress(double lat, double lon);
```

Comment:
Used in AA

Reply:
If this logic is incorrect, I can fix it:
```java
    String address = mMapObject.getAddress();
    if (address.isEmpty())
      address = Framework.nativeGetAddress(mMapObject.getLat(), mMapObject.getLon());
    if (!address.isEmpty())
      builder.addText(address);
```

Reply:
Please raise a PR to master?

Reply:
> Please raise a PR to master?

?